steps/check-your-appeal/content.en.json

```diff
@@ -18,5 +18,17 @@
     "mobile": "Mobile"
   },
   "representative": "Representative",
-  "reasonsForAppealing": "Reasons for appealing"
+  "reasonsForAppealing": "Reasons for appealing",
+  "fields": {
+    "signer": {
+      "error": {
```

Comment:
For consistency please ensure the error object within a field is defined last.

Reply:
done